process_scheduling_algorithms/non_preemptive_priority_scheduling.c

```diff
@@ -0,0 +1,400 @@
+/**
+ * @file
+ * @details
+ * [Non-Preemptive Priority
+ * Scheduling](https://en.wikipedia.org/wiki/Scheduling_(computing)#Priority_scheduling)
+ * is a scheduling algorithm that selects the tasks to execute based on
+ * priority. In this algorithm, processes are executed according to their
+ * priority. The process with the highest priority is to be executed first and
+ * so on. In this algorithm, a variable is maintained known as the time quantum.
+ * The length of the time quantum is decided by the user. The process which is
+ * being executed is interrupted after the expiration of the time quantum and
+ * the next process with the highest priority is executed. This cycle of
+ * interrupting the process after every time quantum and resuming the next
+ * process with the highest priority continues until all the processes have
+ * been executed.
+ * @author [Aryan Raj](https://github.com/aryaraj132)
+ */
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>     /// for IO operations
#include <assert.h>    /// for assert
```